src/repo/zcl_abapgit_dot_abapgit.clas.abap

```diff
@@ -229,17 +203,18 @@ CLASS ZCL_ABAPGIT_DOT_ABAPGIT IMPLEMENTATION.
 
   METHOD get_i18n_langs.
 
-    DATA lt_langs TYPE zif_abapgit_dot_abapgit=>ty_langs_tt.
+    rv_langs = zcl_abapgit_lxe_texts=>convert_table_to_lang_string( ms_data-i18n_languages ).
```

Comment:
... and so this part should happen in pages

Reply:
separate PR.